documentation/de/support_packages/working_with_cookies/index.html

```diff
@@ -11,6 +11,10 @@
     <title>Handling von Cookies :: Dokumentation für Selenium</title>
 
```

Comment:
Hi @boris779 ,

Other docs describe this page as 'working with cookies' and in the translated page, it says 'Handling cookies'. It should be 'Arbeiten mit Cookies' i think.

Reply:
you are right, I will change that